controller/workitem_blackbox_test.go

```diff
@@ -101,6 +108,129 @@ func TestGetWorkItemWithLegacyDescription(t *testing.T) {
 	test.DeleteWorkitemOK(t, nil, nil, controller, *result.Data.ID)
 }
 
+// TestSuccessReorder is positive test which tests successful reorder by providing valid input
+func TestSuccessReorder(t *testing.T) {
```

Comment:
Please stick to the convention of building test function names: Test + What + How (+ Expected).  In other words you should have have named this `TestReorderWorkitemOK`.